### PR TITLE
CLDR-16592 Add Anii (blo-BJ) locale

### PR DESCRIPTION
**Pull request by DavidLRowe**
CLDR-16592

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: common/main/blo_BJ.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<ldml>
+	<version number="$Revision$”/>
+	<identity>
```
Comment (srl295) [inline common/main/blo_BJ.xml:5]:
```suggestion
	<identity>
	<version number="$Revision$”/>
```

File: common/main/blo.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<ldml>
+	<version number="$Revision$”/>
+	<identity>
+		<language type="blo"/>
+	</identity>
```
Comment (srl295) [inline common/main/blo.xml:7]:
```suggestion
	<identity>
	<version number="$Revision$”/>
		<language type="blo"/>
	</identity>
```

**Review by srl295 — DISMISSED** [review]
LG but needs xml fix

**Review by srl295 — APPROVED** [review]

**Comment by DavidLRowe** [comment]
@srl295 Looks as though there's one remaining error:
Error:  (LikelySubtagsTest.java:324)  Error: Missing English translation for: blo
How do I address this?

**Comment by srl295** [comment]
> @srl295 Looks as though there's one remaining error:
> Error: (LikelySubtagsTest.java:324) Error: Missing English translation for: blo
> How do I address this?

@DavidLRowe I'll propose a revision to this PR to disable the test for this language.   until the test is fixed, which is in plan.

**Comment by srl295** [comment]
@DavidLRowe  try this sub-PR into your PR https://github.com/DavidLRowe/cldr/pull/2

**Comment by srl295** [comment]
@DavidLRowe commit message shouldn't block… but you could use the squash button to head that off.

**Comment by DavidLRowe** [comment]
@srl295 Format violation error in the modification to the test

**Comment by srl295** [comment]
@DavidLRowe https://github.com/DavidLRowe/cldr/pull/3

**Comment by DavidLRowe** [comment]
Error is:
blo [Anii]	error	▸Core_Data|Alphabetic_Information|Characters_in_Use|Main_Letters◂	〈[a b c d e f g h i j k l m n o p q r s t u v w x y z]〉	【】	〈[a ǝ b c ɖ e ɛ f g {gb} h i ɩ j k {kp} l m n {ny} ŋ {ŋm} o ɔ p r s {sh} t u ʊ w y \u0301 \u0300 \u0302]〉	«=»	【】	⁅illegal characters in exemplars⁆	❮Error: Should be limited to (specific-script - uppercase - invisibles + İ); thus not contain: ‎[\u0300-\u0302]‎❯

Changing to:
a á à â ǝ {ǝ́} {ǝ̀} {ǝ̂} b c ɖ e é è ê ɛ {ɛ́} {ɛ̀} {ɛ̂} f g {gb} h i í ì î ɩ {ɩ́} {ɩ̀} {ɩ̂} j k {kp} l m ḿ {m̀} n ń ǹ {ny} ŋ {ŋ́} {ŋ̀} {ŋm} o ó ò ô ɔ {ɔ́} {ɔ̀} {ɔ̂} p r s {sh} t u ú ù û ʊ {ʊ́} {ʊ̀} {ʊ̂} w y

**Comment by DavidLRowe** [comment]
@srl295 I think it's ready for review now.

**Review by srl295 — APPROVED** [review]

**Pull request merged**